x/ibc-transfer/spec/06_events.md

```diff
@@ -0,0 +1,45 @@
+<!--
+order: 6
+-->
+
+# Events
+
+## MsgTransfer
+
+| Type         | Attribute Key | Attribute Value |
+|--------------|---------------|-----------------|
+| ibc_transfer | sender        | {sender}        |
+| ibc_transfer | receiver      | {receiver}      |
+| message      | action        | transfer        |
+| message      | module        | transfer        |
+
+## OnRecvPacket callback
+
+| Type                  | Attribute Key | Attribute Value |
+|-----------------------|---------------|-----------------|
+| fungible_token_packet | module        | transfer        |
```

Comment:
should we add a type for each callback? like `transfer_onrecvpacket`? and then use `fungible_token_packet_transfer` for the type `MsgTransfer`?